src/Makefile.am

```diff
@@ -1,15 +1,7 @@
 sbin_PROGRAMS = hulda
 
-INCLUDES = \
-	`pkg-config --cflags libosso` \
-	`pkg-config --cflags glib-2.0` \
-	`pkg-config --cflags gconf-2.0` \
-	`pkg-config --cflags dbus-1` \
-	`pkg-config --cflags dbus-glib-1` \
-	-DLOCALEDIR=\"`pkg-config --variable=localedir osso-af-settings`\"
-
-hulda_LDFLAGS = \
-	`pkg-config --libs gconf-2.0 dbus-glib-1`
+hulda_CFLAGS= $(GLIB_CFLAGS) $(GCONF_CFLAGS) $(DBUS_GLIB_CFLAGS) $(DBUS_CFLAGS)
+hulda_LDADD = $(GLIB_LIBS) $(GCONF_LIBS) $(DBUS_GLIB_LIBS) $(DBUS_LIBS)
 
```

Comment:
You're missing OSSO_CFLAGS/OSSO_LIBS. Also, I don;t see -DLOCALEDIR=\"`pkg-config --variable=localedir osso-af-settings`\" replacement

Reply:
> `pkg-config --variable=localedir osso-af-settings`

Ah, that explains the last build error I had. I re-added that line, as I'm not sure how to put it in `configure.ac` instead. It builds successfully for me now.